pennylane/utils.py

```diff
@@ -432,6 +432,12 @@ def circuit2():
             "None was passed as an argument to inv. "
             + "This could happen if inversion of a template without the template decorator is attempted."
         )
+    elif callable(operation_list):
+        raise ValueError(
+            "A function was passed as an argument to inv. "
+            + "This could happen if inversion of a template function is attempted. "
+            + "Please use inv on the function including its arguments, as in inv(template(args))."
```

Comment:
This is very nice! Minor comment: you don't need the `+` here, multiline string concatenation is implicit in Python if the strings are enclosed within parenthesis

Reply:
Thanks, didn't know that!